src/userspace/exec_monitor.cpp

```diff
@@ -81,4 +104,45 @@ int ExecMonitor::run()
 
 }
 
-int ExecMonitor::export_data() { return 0;} // TODO
\ No newline at end of file
+void ExecMonitor::export_data(process_info *p)
+{
+	if (export_format == STDOUT)
+	{
+		printf("%d\t%d\t%d\t%s\n", p->ppid, p->pid, p->tgid, p->name);
```

Comment:
Since you are writing C++, why not use the C++ methods to write to stdout consistently everywhere ?